optimize/backend/src/main/java/io/camunda/optimize/dto/optimize/importing/DecisionInstanceDto.java

```diff
@@ -43,4 +41,25 @@ public class DecisionInstanceDto implements OptimizeDto {
   private Set<String> matchedRules = new HashSet<>();
   private String engine;
   private String tenantId;
+
+  public static final class Fields {
```

Comment:
❓ How come in some classes we're using a class like this one while in others (eg IdentityLinkLogEntryDto ) we're using an enum?

Reply:
Also replying to @matthewBearCamunda 

Well this comes from how the annotation was used before. In some cases, it had the `asEnum = true` argument, in other cases it didn't have it. The 2 cases have a different vanilla Java code, so the results are different.

I don't know why it was this way originally. CC'ing @RomanJRW to see if he happens to have more context about this